autometa/taxonomy/ncbi.py

```diff
@@ -364,28 +338,81 @@ def parse_merged(self):
             logger.debug('merged loaded')
         return merged
 
+    def get_lineage_names(self, taxid):
+        """
+        Returns the lineage names of the taxid
+
+        Parameters
+        ----------
+        taxid : int
+            identifer for a taxon in the Taxonomy Database by NCBI
+
+        Returns
+        -------
+        set
+            lineage of the taxid
+        """
+        lineage = set()
+        while taxid != 1:
+            if self.rank(taxid) not in NCBI.CANONICAL_RANKS:
+                taxid = self.parent(taxid)
+                continue
+            lineage.add(self.name(taxid))
+            taxid = self.parent(taxid)
+        return lineage
+
     def is_common_ancestor(self, parent_taxid, child_taxid):
-        """Determines whether the provided taxids have a non-root common ancestor
+        """
+        Determines whether the provided taxids have a non-root common ancestor
 
         Parameters
         ----------
         parent_taxid : int
-            Description of parameter `parent_taxid`.
+            identifer for a taxon in the Taxonomy Database by NCBI
         child_taxid : int
-            Description of parameter `child_taxid`.
+            identifer for a taxon in the Taxonomy Database by NCBI
```

Comment:
There should be no expectation that one taxid is the parent and one the child, so I don't think it is appropriate to have the argument names `parent_taxid` and `child_taxid`. It would be better to have `taxid_1` and `taxid_2` or something.